src/sass/components/_footer.scss

```diff
@@ -0,0 +1,151 @@
+.footer {
+  background-color: var(--color-secondary-background);
+
+  &__wrapper {
+    padding-bottom: 43px;
+    border-bottom: 1px solid #e1e1e1;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    @media screen and (min-width: 768px) {
+      flex-direction: row;
+    }
+
+    &_title {
+      font-family: 'Titan One';
```

Comment:
font-family: 'Titan One', cursive; - trzeba określać jeszcze typ czcionki, dzięki temu przy niezaczytaniu tej konkretnej przeglądarka wybierze inną tego samego typu. Później dodam zmienną na to